diff --git a/tests/TestAsync.hs b/tests/TestAsync.hs
index 2a18c1a..a620c8d 100644
--- a/tests/TestAsync.hs
+++ b/tests/TestAsync.hs
@@ -219,7 +219,7 @@ asyncStmTests transport = do
 -- | Given a @builder@ function, make and run a test suite on a single transport
 testMain :: (NT.Transport -> IO [Test]) -> IO ()
 testMain builder = do
-  Right (transport, _) <- createTransportExposeInternals "127.0.0.1" "0" (\sn -> ("127.0.0.1", sn)) defaultTCPParameters
+  Right (transport, _) <- createTransportExposeInternals (defaultTCPAddr "127.0.0.1" "0") defaultTCPParameters
   testData <- builder transport
   defaultMain testData
 
